server/test/suites/categoryRouter.test.js

```diff
@@ -0,0 +1,169 @@
+// MongoDB Models
+const Category = require('../../db/models/category');
+const User = require('../../db/models/user');
+const Item = require('../../db/models/item');
+
+// Require Testing Tools
+const server = require('../../index');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+
+// Import MongoDB-connecting functions
+const { getMongoDB } = require('../../db');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const mongoose = require('mongoose');
+
+// Set up Chai
+const expect = chai.expect;
+chai.use(chaiHttp);
+
+// Mock documents used for testing
+let item1;
+let item2;
+let testUser1;
+let testUser2;
+let testCategory1;
+let testCategory2;
+let testCategory3;
```

Comment:
I left these as let and not const bc they get assigned a value within the before() function. Is that okay? @ricazhang

Reply:
Sounds good!